frontend/src/pages/learningCenter/LearningCenterDataView.tsx

```diff
@@ -94,9 +98,10 @@ const LearningCenterDataView: React.FC<LearningCenterDataViewProps> = React.memo
       return (
         <div className={listViewClasses} aria-label="Simple data list example">
           <LearningCenterListHeader />
-          {filteredDocApps.map((doc) => (
+          {filteredDocApps.map((doc, idx) => (
             <OdhDocListItem
               key={`${doc.metadata.name}`}
+              itemId={`${doc.spec.appDisplayName}-${idx}`}
```

Comment:
Yeah -- okay, took forever to fix my server and get the test in a place I could verify this.

Error from React:
```
Encountered two children with the same key, `jupyter-doc`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.
```

I can cause this error due to the way the information is processed. We'll need to get the `${type}-${name}` going on for key as well. 

Side note: It does render both cards, just with the error.
![image](https://user-images.githubusercontent.com/8126518/221004669-1a93af2d-cccb-457f-adba-31abec096570.png)

Reply:
Nice catch @andrewballantyne , updated based on your feedback! Let me know if these changes [here](https://github.com/opendatahub-io/odh-dashboard/pull/970/commits/37069823facde07bc51d1b4709a177cebe21810f) are what you had in mind.